webrender/src/clip_scroll_node.rs

```diff
@@ -46,9 +46,26 @@ impl StickyFrameInfo {
 }
 
 #[derive(Debug)]
-pub enum NodeType {
+pub enum PositioningNodeType {
```

Comment:
possibly alternative names: "SpatialNode", "PositionFrame"

Reply:
Renamed to Spatial :+1: